homeassistant/components/aladdin_connect/sensor.py

```diff
@@ -40,7 +40,6 @@ class AccSensorEntityDescription(
 SENSORS: tuple[AccSensorEntityDescription, ...] = (
     AccSensorEntityDescription(
         key="battery_level",
-        name="Battery level",
```

Comment:
Changing the name is a "breaking" change isn't it? - Should this have a translation key keeping it the same?  I'm not against using the "standard" name and translation, just need to mark it breaking it that's the case.

Reply:
afaik, the entity is still identified by the same unique id, so in UI etc I think it will adapt to the new name automatically. The only thing I am now thinking of are the YAML configured things. Now that I think of it I am not sure how they handle it

Reply:
Correct, the name change isn't breaking. If a user sets their own name, that overrides always (and doesn't change).

Reply:
Also now curious, how does YAML based stuff (generic thermostat, template entities) handle that?